ddprof-lib/build.gradle

```diff
@@ -157,8 +160,62 @@ def initSubrepoTask = tasks.register('initSubrepo', Copy) {
   into "${projectDir}/src/main/cpp-external"
 }
 
-initSubrepoTask.configure {
-  dependsOn cloneAPTask
+def patchStackFrame = tasks.register("patchStackFrame") {
+  description = 'Patch stackFrame_x64.cpp after copying'
+  configure {
+    dependsOn copyUpstreamFiles
+  }
+  doLast {
+    def file = file("${projectDir}/src/main/cpp-external/stackFrame_x64.cpp")
+    if (!file.exists()) throw new GradleException("File not found: ${file}")
+
+    def content = file.getText('UTF-8')
+    def original = content
+
+    // Add no_sanitize to unwindStub
```

Comment:
why not add compiler directives ? because this is upstream I assume

Reply:
Yep. It's upstream.
I am not thrilled by this but the find-replace turned out to be the least intrusive measure - I will try proposing to add the directive upstream. Ideally, we should not be patching the upstream code.